lib/services/OrderService.js

```diff
@@ -0,0 +1,25 @@
+import dotenv from 'dotenv';
+import Order from '../models/Order';
+import { sendSms } from '../utils/twilio.js';
+dotenv.config();
```

Comment:
normally you'll want to run this right after you import it (line two). Also looked up the docs and you can do this wizardry 
```
import "dotenv/config.js";
```